manifests/o/OpenJDK16/AdoptOpenJDK/16.0.1.9/OpenJDK16.AdoptOpenJDK.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: OpenJDK16.AdoptOpenJDK

```

Comment:
This is a duplicate of AdoptOpenJDK.OpenJDK.16, as such, it shouldn't be merged.